ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource

```diff
@@ -28,12 +28,11 @@ Upload Serving Runtime Template
     [Documentation]    Uploads via UI a YAML file containing a custom Serving Runtime definition
     [Arguments]    ${runtime_filepath}    ${serving_platform}
     Click Add Serving Runtime Template Button
-    SeleniumLibrary.Click Element   ${UPLOAD_RUNTIME_BTN_XP}
+    Element Should Be Enabled   ${UPLOAD_RUNTIME_BTN_XP}
```

Comment:
don't we need to click?

Reply:
Apparently clicking on it now breaks things, as a dialog of the browser is opened and not closed, so it's better to verify the button is enabled, and then upload the file directly with:
`SeleniumLibrary.Choose File    ${EDITOR_RUNTIME_BTN_XP}//input[@type="file"]  ${pwd}/${runtime_filepath}`

Reply:
oke :+1: